packages/docusaurus-plugin-content-docs/package.json

```diff
@@ -5,7 +5,9 @@
   "main": "lib/index.js",
   "types": "src/plugin-content-docs.d.ts",
   "scripts": {
-    "build": "tsc",
+    "build": "tsc && yarn build-theme-esm && yarn build-client-esm",
+    "build-theme-esm": "esbuild src/theme/hooks/useDocs.ts > lib/theme/hooks/useDocs.js",
+    "build-client-esm": "esbuild src/client/docsClientUtils.ts > lib/client/docsClientUtils.js",
```

Comment:
Somehow esbuild doesn't like that the client files are not in esm, so I setup the script to compile the client files to esm.
This fixed most of the website loading issue. However still cannot get the live code block working :( (See [this](https://deploy-preview-4487--docusaurus-2.netlify.app/docs/markdown-features/code-blocks#interactive-code-editor))